docs/articles/documentation/test-api/built-in-waiting-mechanisms.md

```diff
@@ -37,7 +43,7 @@ If the element does not appear, the test will fail.
 
 The exceptions are [t.setFilesToUpload](actions/upload.md#populate-file-upload-input) and [t.clearUpload](actions/upload.md#clear-file-upload-input) actions that do not require that the target element is visible.
```

Comment:
The [t.setFilesToUpload](actions/upload.md#populate-file-upload-input) and [t.clearUpload](actions/upload.md#clear-file-upload-input) actions are exceptions because they do not require a visible target element.